folium/map.py

```diff
@@ -363,11 +363,8 @@ def __init__(self, location, popup=None, icon=None):
         ----------
         location: tuple or list, default None
             Latitude and Longitude of Marker (Northing, Easting)
-        popup: string or tuple, default 'Pop Text'
-            Input text or visualization for object. Can pass either text,
-            or a tuple of the form (Vincent object, 'vis_path.json')
-            It is possible to adjust the width of text/HTML popups
-            using the optional keywords `popup_width` (default is 300px).
```

Comment:
I like the slim docstring version :+1: 

However, it is not clear that people can change the `popup_width` from the Marker class.  (I am OK with that.)